src/Microsoft.Health.Dicom.Core/Features/Workitem/Model/WorkitemDetail.cs

```diff
@@ -0,0 +1,18 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+namespace Microsoft.Health.Dicom.Core.Features.Workitem.Model
+{
+    public sealed class WorkitemDetail
```

Comment:
Can it be WorkitemStoreEntry? or WorkitemEntry?

Reply:
WorkitemStoreEntry? @josiahvinson do you have a preference?

Reply:
Done